controllers/configurationpolicy_controller.go

```diff
@@ -2826,11 +2826,14 @@ func (r *ConfigurationPolicyReconciler) updatePolicyStatus(
 			eventType = eventWarning
 		}
 
+		eventMessage := fmt.Sprintf("%s%s", policy.Status.ComplianceState, msg)
+		log.Info("Policy status message", "status", eventMessage)
```

Comment:
The logger here doesn't automatically include the policy name. Could you please include that as a key/value pair?

Reply:
Good point.  Now it has the policy name.  Thanks